arangod/Cluster/AutoRebalance.cpp

```diff
@@ -230,29 +230,36 @@ ShardImbalance AutoRebalanceProblem::computeShardImbalance() const {
 
 std::vector<double> AutoRebalanceProblem::piCoefficients(
     Collection const& c) const {
-  std::vector<double> tmp;
-  tmp.resize(dbServers.size(), 0);
+  std::vector<double> leaders;
+  leaders.resize(dbServers.size(), 0);
+  std::vector<bool> haveShards;
+  haveShards.resize(dbServers.size(), 0);
```

Comment:
For improved readability, I guess we should set each member to `false` in a vector<bool> and not to `0`.
```suggestion
  haveShards.resize(dbServers.size(), false);
```